tests/test_bgp_af.rb

```diff
@@ -828,7 +1117,15 @@ def network_cmd(af, dbg)
                  "#{dbg} Test 3. Restore the removed networks")
 
     # Test: Change route-maps on existing networks
-    should = master.map { |network, rm| [network, "#{rm}_55"] }
+    if platform == :ios_xr
+      %w(rtmap1_55, rtmap2_55, rtmap3_55, rtmap5_55,
```

Comment:
Note that commas are not needed when using the %w syntax - this would be more correctly written as
```ruby
%w(rtmap1_55 rtmap2_55 rtmap3_55 ...
```
Same comment elsewhere in this file.